source/use-cases/client-side-field-level-encryption-local-key-to-kms.txt

```diff
@@ -0,0 +1,26 @@
+==================================================
+Client-Side Field Level Encryption: Convert to KMS
+==================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _fle-convert-to-a-remote-master-key:
+
+Convert to a Remote Master Key
+------------------------------
+
```

Comment:
While we have a lead-in into this document from the CSFLE Guide, I think it would be helpful to include a couple sentences about who should read the guide, stage of implementation (ready for production), a reference to the CSFLE guide (in case someone lands on the wrong page), and what the benefits of a KMS are.

Reply:
Another item important to mention: any data that was inserted with the old data encryption key needs to be migrated or else it will not be decrypt-able.